frontend/src/app/my-courses/course/office-hours/office-hours-editor/office-hours-editor.component.ts

```diff
@@ -59,15 +56,23 @@ export class OfficeHoursEditorComponent {
   /* Holds the virtual room. */
   virtualRoom: Room | undefined;
 
+  public days: { [day: string]: boolean } = {
+    Mon: false,
```

Comment:
Making DoWs as an enum might help to prevent future issues using strings as keys.